pkg/cmd/cluster/cluster_link.go

```diff
@@ -95,7 +95,13 @@ func newCmdClusterLink(ctx api.Context) *cobra.Command {
 			}
 
 			attachedClient := linker.New(ctx.HTTPClient(attachedCluster), ctx.Logger())
-			return attachedClient.Link(linkRequest)
+			ctx.Logger().Info("Linking the cluster...")
```

Comment:
Could this be done from `pkg/cluster/linker/linker.go` inside the Link method instead?